platform/linux-generic/pktio/dpdk.c

```diff
@@ -160,6 +160,9 @@ typedef struct {
 typedef struct ODP_ALIGNED_CACHE {
 	/* --- Fast path data --- */
 
+	/* Function for mbuf to ODP packet conversion */
+	int (*mbuf_to_pkt_fn)(pktio_entry_t *pktio_entry, odp_packet_t pkt_table[],
```

Comment:
Good idea, could be worth considering similar treatment in other packet I/O drivers as well at some point.